spec/lib/mail_handler/mail_handler_spec.rb

```diff
@@ -516,6 +516,26 @@ def expect_header_string(fixture_file, header, header_string)
       attr.should == expected_attributes[index]
     end
   end
+
+    describe :get_attachment_attributes do
+
+    it 'should handle an Outlook attachment with HTML generated from RTF' do
+      mail = get_fixture_mail('outlook-encoding-rtf.email')
+      attribute_hashes = MailHandler.get_attachment_attributes(mail)
+      attribute_hashes.each do |attribute_hash|
+        attribute_hash[:body].valid_encoding?.should be_true
+      end
+    end
+
+    it 'should handle an Outlook attachment with multiple attachments' do
+      mail = get_fixture_mail('outlook-encoding-multiple.email')
+      attribute_hashes = MailHandler.get_attachment_attributes(mail)
+      attribute_hashes.each do |attribute_hash|
+        attribute_hash[:body].valid_encoding?.should be_true
+      end
+    end
+
+  end
```

Comment:
`end` at 538, 2 is not aligned with `describe :get_attachment_attributes do` at 520, 4.